workflows/src/main/kotlin/com/r3/developers/apples/workflows/PackageApples.kt

```diff
@@ -0,0 +1,66 @@
+package com.r3.developers.apples.workflows
+
+import com.r3.developers.apples.contracts.AppleCommands
+import com.r3.developers.apples.states.BasketOfApples
+import net.corda.v5.application.flows.CordaInject
+import net.corda.v5.application.flows.ClientRequestBody
+import net.corda.v5.application.flows.ClientStartableFlow
+import net.corda.v5.application.marshalling.JsonMarshallingService
+import net.corda.v5.application.membership.MemberLookup
+import net.corda.v5.base.annotations.Suspendable
+import net.corda.v5.ledger.common.NotaryLookup
+import net.corda.v5.ledger.utxo.UtxoLedgerService
+import java.time.Instant
+import java.time.temporal.ChronoUnit
+
+class PackageApplesFlow : ClientStartableFlow {
+
+    private data class PackApplesRequest(val appleDescription: String, val weight: Int)
+
+    @CordaInject
+    lateinit var jsonMarshallingService: JsonMarshallingService
+
+    @CordaInject
+    lateinit var memberLookup: MemberLookup
+
+    @CordaInject
+    lateinit var notaryLookup: NotaryLookup
+
+    @CordaInject
+    lateinit var utxoLedgerService: UtxoLedgerService
+
+    @Suspendable
+    override fun call(requestBody: ClientRequestBody): String {
+        val request = requestBody.getRequestBodyAs(jsonMarshallingService, PackApplesRequest::class.java)
+        val appleDescription = request.appleDescription
+        val weight = request.weight
+        val notary = notaryLookup.notaryServices.single()
+        val myKey = memberLookup.myInfo().ledgerKeys.first()
+
+        // Building the output BasketOfApples state
+        val basket = BasketOfApples(
```

Comment:
Why not rename `BasketOfApples` => `ApplesBasket` ?

Reply:
For now, I will follow what is described in [Building your first Cordapp > Write States](https://docs.r3.com/en/platform/corda/5.0/developing-applications/basic-cordapp/state.html#check-your-work) so that it matches the tutorial, 

I am happy to change it if we agree so in the future PR when I update the solution to an improved standard

Reply:
If we change both documentation (totally possible) and the samples code I think it's worthy ... 
Please follow-up with a new JIRA and update the docs requirements tab of that new JIRA issue such that @colin-hillis and his team are aware

Reply:
https://r3-cev.atlassian.net/browse/CORE-15314 - kotlint
https://r3-cev.atlassian.net/browse/CORE-15315 - java

Will do. I'll be in touch with colin's team too!